### PR TITLE
protonmail is indeed compatible with OpenPGP, just not with Autocrypt.

### PR DESCRIPTION
**Pull request by missytake**
We received some feedback that the FAQ is slightly incorrect - this should cover it :)

### CODE REVIEW

**Comment by missytake** [comment]
It might make sense to directly include #466 as well in this PR... maybe tomorrow^^

**Comment by missytake** [comment]
btw, see https://staging.delta.chat/468/en/help#is-delta-chat-compatible-with-protonmail--tutanota--criptext for preview.

File: en/help.md [hunk 1]
```diff
@@ -456,10 +456,15 @@ Otherwise, deleting messages or multi-device setups might not work properly.
...
 - In any case you can use Delta Chat to send Messages to people who use
   Protonmail, Tutanota, or Criptext. Those messages will not be End-to-End
-  encrypted, though. The End-to-End encryption those providers offer is only
-  working inside their platforms, and not compatible with anyone outside.
+  encrypted, though. The End-to-End encryption those providers offer is not 
+  compatible with [Autocrypt](https://autocrypt.org/), the standard Delta Chat
+  uses.
 - Delta Chat can e2e-encrypt through any e-mail provider with any
   [Autocrypt-enabled e-mail app](https://autocrypt.org/dev-status.html).
+- Users who pay for Protonmail Desktop might be able to use the IMAP Bridge for
```
Comment (r10s) [inline en/help.md]: i think, this is much too much text in an _f_aq (that also goes to offline help) for an provider that is actually flagged as being broken.

Comment (r10s) [reply en/help.md]: maybe say:
- No, you cannot simply log in to Delta Chat using your Protonmail, Tutanota or Criptext account; they do not support open standards for that.
- Yes, you can use Delta Chat to send Messages to people who use these providers. Again, due to a lack of supported standards, these mails are not end-to-end encrypted.

**Comment by r10s** [comment]
just came over https://github.com/ProtonMail/proton-bridge/issues/220 - so, i think, treating protonmail as broken is even more reasonable.

**Comment by missytake** [comment]
(thanks for pushing this^^)

**Review by r10s — APPROVED** [review]
still not super-happy with the framing, but it is more correct now, so let's move forward :)

**Pull request merged**